third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -32,9 +33,15 @@
 #include <google_smart_card_common/pp_var_utils/construction.h>
 #include <google_smart_card_common/pp_var_utils/extraction.h>
 
-// This arbitrary chosen constant is used as a stub for the device bus number
-// (as the chrome.usb API does not provide means of retrieving this).
-const uint8_t kFakeDeviceBusNumber = 42;
+//
+// We use stubs for the device bus number (as the chrome.usb API does not provide means of 
+// retrieving it). We modify this for a device when opening the device fails.
+// This makes PCSC recognize it as a new device which causes PCSC to retry opening it.
+//
+
+std::unordered_map<int64_t, uint8_t> kBusNumbers;
```

Comment:
The style guide forbids global objects with nontrivial destructors: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables
Can we make it a member of the LibusbOverChromeUsb?

Also the name looks like if it were a constant, while it's not.

Reply:
Done